code/LINDA/LINDA_turf_tile.dm

```diff
@@ -311,7 +311,7 @@
 /atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	if(last_forced_movement >= SSair.times_fired)
 		return 0
-	else if(!anchored && !pulledby)
+	else if((!anchored || move_resist != INFINITY) && !pulledby)
```

Comment:
What this conditional means,
If you are (not anchored OR don't have infinite move_resist) and aren't being pulled by anyone

Then be affected by atmos

What this means is that in order to not be affected by atmos, you need to both be `anchored` and have infinite `move_resist`

What you actually want is:
`else if(!(anchored || move_resist == INFINITY) && !pulledby)`
This way, this will evaluate to false if you are `anchored` or have infinite `move_resist`.

Reply:
Aye I figured this out after the bug started. I chose to make it into 2 if statements now to increase readability a bit. And because I wanted to make sure I didn't fuck up the logic again.